src/ert/storage/local_ensemble.py

```diff
@@ -562,6 +562,15 @@ def _load_parameters_lazy(
         df = pl.scan_parquet(group_path)
         return df
 
+    def load_all_scalar_keys(self, transformed: bool = False) -> pl.DataFrame:
```

Comment:
not sure if we need this function. There is already a function called load_scalars
https://github.com/equinor/ert/blob/38adc63a90aca4ff626cc199610a0518e89acf02/src/ert/storage/local_ensemble.py#L688

Reply:
Ill refactor the existing one